.travis.yml

```diff
@@ -40,9 +40,12 @@ env:
 
 python:
   - "2.7"
-  - "3.3"
-  - "3.4"
   - "3.5"
+  - "pypy"
+
+matrix:
+  allow_failures:
+    - python: pypy
 
```

Comment:
`pypy` should be there only after the Invenio 3 demo site  runs on pypy... Please delete here and in `setup.py` later, squash and repush.